nano/node/json_handler.cpp

```diff
@@ -375,6 +375,24 @@ double nano::json_handler::multiplier_optional_impl (uint64_t & difficulty)
 	return multiplier;
 }
 
+nano::work_version nano::json_handler::work_version_optional_impl (nano::work_version const default_a)
+{
+	nano::work_version result = default_a;
+	boost::optional<std::string> version_text (request.get_optional<std::string> ("version"));
+	if (!ec && version_text.is_initialized ())
+	{
+		if (*version_text == "work_1")
```

Comment:
Could remove duplication & have type safety with: `nano::to_string (nano::work::work_1);`. However It will be more expensive as `std::string` doesn't have a trivial constructor so can't be elided (at least until C++20 has `std::string` constexpr constructors). If this is an issue (it's part of RPC code so don't think so), could make `to_string` a constexpr function and return `const char*`, or perhaps create a new `to_c_string ()` if a `std::string` return arg is still desired in some places.

Reply:
Done, thanks. As you said it's the RPC so just went with the first suggestion 👍